tests/cli/showrev/test_system.py

```diff
@@ -147,19 +158,18 @@ def test_get_slurm_version_kubernetes_max_retry_error(self):
 
     def test_get_site_data_from_s3(self):
         """Test get_site_data downloads from S3."""
-        sitefile = '/opt/cray/etc/site_info.yml'
-        mock_s3 = mock.patch('sat.cli.showrev.system.get_s3_resource').start().return_value
-        mock.patch('sat.cli.showrev.system.get_config_value', return_value='sat').start()
-        mock_open = mock.patch('builtins.open').start()
-        mock_yaml_load = mock.patch('sat.cli.showrev.system.yaml.safe_load').start()
-        sat.cli.showrev.system.get_site_data(sitefile)
-        mock_s3.Object.assert_called_once_with('sat', sitefile)
-        mock_s3.Object.return_value.download_file.assert_called_once_with(sitefile)
-        mock_open.assert_called_once_with(sitefile, 'r')
-        mock_open.return_value.__enter__.return_value.read.assert_called_once_with()
-        mock_yaml_load.assert_called_once_with(
-            mock_open.return_value.__enter__.return_value.read.return_value
-        )
+        sat.cli.showrev.system.get_site_data(self.sitefile)
+
+        self.mock_s3.Object.assert_called_once_with('sat', self.sitefile)
+        self.mock_s3.Object.return_value.download_file.assert_called_once_with(self.sitefile)
+        self.mock_open.assert_called_once_with(self.sitefile, 'r')
+        self.mock_yaml_load.assert_called_once_with(self.mock_file_obj)
+
+    def test_get_site_data_sitefile_empty(self):
+        """Test that an error is logged if the sitefile is empty."""
```

Comment:
Currently, it's a warning.

Reply:
With the change above, this description should be correct.